README.md

```diff
@@ -21,21 +21,25 @@ docker run --env-file .env --rm -p 8080:8080 srcd/code-annotation
 
 ### Non-docker
 
-```bash
-go get github.com/src-d/code-annotation/...
-cd $GOPATH/github.com/src-d/code-annotation
-make serve
-```
+Download binary from [releases](releases) for you platform.
 
 ## Development
 
-Backend:
+Global dependencies:
```

Comment:
This information should go int he `CONTRIBUTING.md` this is not the place for this.

Reply:
yes. We have https://github.com/src-d/code-annotation/pull/29 for it.
I'll rebase later.